src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -3116,9 +3117,13 @@ class CgroupUtils(object):
                 i += 4
                 continue
             if buf[i] == '-':
-                ret += '/'
+                ret += b'/'
             elif buf[i].isalnum() or buf[i] in '_.':
-                ret += buf[i]
+                if (sys.version_info[0] < 3):
+                    ret += buf[i]
+                else:
+                    # normal char -- should never require UTF-8; if it happens, exception
+                    ret += buf[i].encode('ascii','strict')
```

Comment:
Nitpick - Please add a space after the comma.

Reply:
Consider it done.

Reply:
Done, committed.